exporters/scss/src/js/constants/weight.ts

```diff
@@ -0,0 +1,26 @@
+export type DefaultWeightsType = {
+  thin: number;
+  extralight: number;
+  light: number;
+  normal: number;
+  regular: number;
+  medium: number;
+  semibold: number;
+  bold: number;
+  extrabold: number;
+  black: number;
+  extrablack: number;
+};
+export const defaultWeights: DefaultWeightsType = {
```

Comment:
```suggestion
export const defaultWeights = {
```

The type here is not needed. The type of this object is exact and Typescript can infer it itself.